avocado/core/varianter.py

```diff
@@ -365,15 +365,15 @@ def iteritems(self):
                 yield (leaf.environment_origin[key].path, key, value)
 
 
-def _report_mux_already_parsed(self, *args, **kwargs):
+def _report_variants_already_parsed(self, *args, **kwargs):     # used as member function (self) pylint: disable=W0613
     """
     Raises exception describing that `self.data` alteration is restricted
     """
-    raise RuntimeError("Mux already parsed, altering is restricted. %s %s"
-                       % (args, kwargs))
+    raise RuntimeError("Varianter already parsed, altering is restricted. %s "
+                       "%s" % (args, kwargs))
 
 
-class Mux(object):
+class Varianter(object):
```

Comment:
This is just the rename part. You can see how `Mux` (or now `Varianter`) becomes just an API to register various variants plugins.